setup.py

```diff
@@ -3,27 +3,27 @@
 setup(
     name="dissect",
     install_requires=[
-        "dissect.cim==3.2",
-        "dissect.clfs==1.1",
-        "dissect.cstruct==3.2",
-        "dissect.esedb==3.2",
-        "dissect.etl==3.1",
-        "dissect.eventlog==3.1",
-        "dissect.evidence==3.1",
-        "dissect.extfs==3.1",
-        "dissect.fat==3.1",
-        "dissect.ffs==3.1",
-        "dissect.hypervisor==3.2",
-        "dissect.ntfs==3.1",
-        "dissect.ole==3.1",
-        "dissect.regf==3.1",
-        "dissect.shellitem==3.1",
-        "dissect.sql==3.1",
-        "dissect.target[full]==3.3",
```

Comment:
You lost the `[full]` specifier, is that correct?

Reply:
No, that is not correct